packages/controls/src/controls/group/group.test.ts

```diff
@@ -0,0 +1,421 @@
+import { ControlDataTypeKey } from '../../common'
```

Comment:
Basically existing `ShapeV2` tests, not shown as a `git` rename because I kept `shape-v2.test.ts` with [its own small set of new tests](https://github.com/makeswift/makeswift/pull/909/files?diff=split&w=1#diff-cbe43da181fb2bb2cbafcb9520274fdd4d01671e5880fe635adb1a353526a07eR33).